assembly/src/tokens/mod.rs

```diff
@@ -201,7 +216,12 @@ impl<'a> Token<'a> {
         assert_eq!(Self::IF, self.parts[0], "not an if");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                    ParsingError::missing_param(
+                        self,
+                        "if <true>"
+                    )
```

Comment:
The correct format here would be `if.true`.

Reply:
Changed.